File: src/main/java/fr/vlegall/sochief/repository/RecipeCategoryRepository.java
package fr.vlegall.sochief.repository;

import fr.vlegall.sochief.model.recipe.RecipeCategory;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface RecipeCategoryRepository extends JpaRepository<RecipeCategory, Long> {
    List<RecipeCategory> findAllByOrderByIdAsc();
}

File: src/main/kotlin/fr/vlegall/sochief/repository/RecipeCategoryRepository.kt
package fr.vlegall.sochief.repository

import fr.vlegall.sochief.model.recipe.RecipeCategory
import org.springframework.data.jpa.repository.JpaRepository

interface RecipeCategoryRepository : JpaRepository<RecipeCategory, Long> {
    fun findAllByOrderByIdAsc(): List<RecipeCategory>
}
